### PR TITLE
Enable Checkable Menu Item

### PR DESCRIPTION
**Pull request by QilongTang**
- Add a known issue section
- Enable checkable Menu item following workspace Refenrences as example

TuneUpCheckableMenuItem

### CODE REVIEW

File: TuneUpTests/TuneUpTests.csproj [hunk 1]
```diff
@@ -77,16 +65,16 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="DynamoVisualProgramming.Core">
-      <Version>2.4.0.6186</Version>
+      <Version>2.5.0.7432</Version>
```
Comment (QilongTang) [inline TuneUpTests/TuneUpTests.csproj]: These were wrong version before

File: TuneUpTests/TuneUpTests.csproj [hunk 2]
```diff
@@ -51,19 +51,7 @@
...
-      <HintPath>..\..\Dynamo\bin\AnyCPU\Debug\SystemTestServices.dll</HintPath>
-    </Reference>
-    <Reference Include="TestServices, Version=2.5.0.6450, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\Dynamo\bin\AnyCPU\Debug\TestServices.dll</HintPath>
-    </Reference>
     <Reference Include="WindowsBase" />
-    <Reference Include="WorkspaceDependencyViewExtension, Version=2.5.0.6246, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\Dynamo\bin\AnyCPU\Debug\WorkspaceDependencyViewExtension.dll</HintPath>
```
Comment (QilongTang) [inline TuneUpTests/TuneUpTests.csproj]: Remove these local dependencies for now

Comment (reddyashish) [reply TuneUpTests/TuneUpTests.csproj]: Will we need them in future?

**Comment by reddyashish** [comment]
Looks good to me @QilongTang. I do not have permissions to add labels to PR's of this repo.

File: TuneUpTests/TuneUpTests.csproj (hunk 2 above)
Comment (QilongTang) [reply TuneUpTests/TuneUpTests.csproj]: I dont think we TuneUp currently is depending on workspace References extension, the other two test binaries are included in our DynamoVisualProgramming.Test.Nuget

**Pull request merged**